tools/depcheck/pkg/graph/graph.go

```diff
@@ -75,8 +75,10 @@ func (g *MutableDirectedGraph) NodeByName(name string) (graph.Node, bool) {
 
 // PruneOrphans recursively removes nodes with no inbound edges.
 // Nodes marked as root nodes are ignored.
+// Nodes whose uniqueName is contained within
+// an ignore slice of strings are ignored.
 // Returns a list of recursively pruned nodes.
-func (g *MutableDirectedGraph) PruneOrphans() []*Node {
+func (g *MutableDirectedGraph) PruneOrphans(ignore []string) []*Node {
```

Comment:
Well this came back with a vengeance.  Does our graph not have edges from the rootNodes to their first level deps or something?  Lack of an example anywhere in this pull makes it impossible to offer alternatives.